### DISCUSSION TITLE
Opens Google and a few other page in desktop mode

### DISCUSSION THREAD

**Discussion by webbb82** [bromite/bromite#2530] [Q&A]
I really love bromine, and use that as my default browser but this is really annoying that I have to zoom in every time I open Google or every other website. It will do it and I don't know why I was just was curious if anyone else has that issue and no effects for it
I tried to click desktop mode and turn it off again but that did nothing

**Comment by Universalizer** [comment]
**1) Menu > Settings > Site settings > Desktop site > Toggle On or Off (General Settings for Chromium / Bromite).**

2) Additionally review for to > Check or uncheck other optional settings.

3) At ‘always request desktop site’ it's right side ; click on ‘drop-down arrow button’ for to expand (Above ADD SITE EXCEPTION), under it probably contains list of sites, plus all above Top of this settings page ; their is ‘Search bar icon’ click it to type and find ; site name to be filter.

4) Click it, that pop-ups window prompts > 'SELECT' 1) either Desktop site or Mobile site. 2) Or Selected option to completely Remove it.

5) Come to that site or webpage and refresh it.

_Additional Information :-_
_6) Their are more two ways to set this preference, but this above mentioned way is best to practice._

_Hopefully helpful._

**Reply by futurk** [reply]
> **1) Menu > Settings > Site settings > Desktop site > Toggle On or Off (General Settings for Chromium / Bromite).**
> 
> 2. Additionally review for to > Check or uncheck other optional settings.
> 3. At ‘always request desktop site’ it's right side ; click on ‘drop-down arrow button’ for to expand (Above ADD SITE EXCEPTION), under it probably contains list of sites, plus all above Top of this settings page ; their is ‘Search bar icon’ click it to type and find ; site name to be filter.
> 4. Click it, that pop-ups window prompts > 'SELECT' 1) either Desktop site or Mobile site. 2) Or Selected option to completely Remove it.
> 5. Come to that site or webpage and refresh it.
> 
> _Additional Information :-_ _6) Their are more two ways to set this preference, but this above mentioned way is best to practice._
> 
> _Hopefully helpful._

"Additionally review for to > Check or uncheck other optional settings."
I didn't really understand this one. There is no such setting over there
![IMG_20230315_204913_425](https://user-images.githubusercontent.com/10360643/225398553-17fbe2e2-70c9-42be-b0ab-70a1892b5348.jpg)

**Comment by Universalizer** [comment]
![©Universalizer®Screenshot_20230316-HHMMSS_Browser Md3](https://user-images.githubusercontent.com/116723165/225423191-85bb12e8-4799-47ad-87e6-28234ff4571c.png)

**Reply by futurk** [reply]
This is really interesting. I don't have those settings:

![Screenshot_20230315-224418_Bromite](https://user-images.githubusercontent.com/10360643/225425725-3db47197-3235-485e-a8e2-97c849cb0d6f.png)
![IMG_20230315_224509](https://user-images.githubusercontent.com/10360643/225425720-118adc87-ed12-4d44-9431-a0ec4baeefd1.jpg)

Can you tell me what is the version of your browser?

**Comment by Universalizer** [comment]
Address bar or omnibox ; type `chrome://flags` and enter it.

Search for ALL flags related to "Desktop" under chrome flags, optional settings.

**Reply by futurk** [reply]
Just enabled "Per-site setting to request desktop site on Android." and ta-da!

![Screenshot_20230315-230043_Bromite](https://user-images.githubusercontent.com/10360643/225429210-cfab8f3d-2fb7-488d-bce7-115e4071d4f2.png)
![Screenshot_20230315-225949_Bromite](https://user-images.githubusercontent.com/10360643/225429216-7f722d51-2b06-42fe-8d40-405145d7536e.png)

Exactly what I was looking for.
Thanks a lot!

**Reply by late2the420party** [reply]
> Address bar or omnibox ; type `chrome://flags` and enter it.
> 
> Search for ALL flags related to "Desktop" under chrome flags, optional settings.

You're a lifesaver. Thanks a lot <33

**Comment by Universalizer** [comment]
👍
